example/algorithms_evaluation/minimal_trap_spaces.py

```
@@ -0,0 +1,15 @@
+import sys
+
+from biodivine_aeon import *
+
+bn = BooleanNetwork.from_file(sys.argv[1])
+bn = bn.infer_valid_graph()
+
+ctx = SymbolicSpaceContext(bn)
+stg = AsynchronousGraph(bn, ctx)
+
+traps = TrapSpaces.minimal_symbolic(ctx, stg)
```

Ruff (0.12.2)

`TrapSpaces` may be undefined, or defined from star imports

(F405)